### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by AliakseiBabich**

### CODE REVIEW

File: src/components/Main/NewSurveyPage/index.js [hunk 1]
```diff
@@ -248,16 +272,40 @@ const NewSurveyPage = props => {
...
+  const pageToShow = surveyState.pages.length ? (
+    <div className={`newSurveyPage-newsurvey__survey-page`}>
+      {addedQuestions}
+      {newQuestion}
+    </div>
+  ) : null;
+
+  const handlePageClick = e => {
+    e.preventDefault();
+    const pageIndex = +e.target.innerText.split(' ')[1] - 1;
```
Comment (maryia-kabash) [inline src/components/Main/NewSurveyPage/index.js]: Error-prone code, and also it is not clear what it does at first sight

File: src/components/Main/NewSurveyPage/index.js [hunk 2]
```diff
@@ -248,16 +272,40 @@ const NewSurveyPage = props => {
...
+    let pageBtnClass = 'newSurveyPage-newsurvey__pageBtn';
+    if (pageState === i) {
+      pageBtnClass = 'newSurveyPage-newsurvey__pageBtn pageBtn--active';
+    }
+    return (
+      <Button
+        className={pageBtnClass}
+        key={i}
+        value={`Cтраница ${i + 1}`}
+        onClick={handlePageClick}
```
Comment (maryia-kabash) [inline src/components/Main/NewSurveyPage/index.js]: data-number="${i}"
handlePageClick(e) => e.target.dataset

Comment (maryia-kabash) [reply src/components/Main/NewSurveyPage/index.js]: onClick={handlePageClick(i)}
const handlePageClick = i=> e => {}

File: src/components/Main/NewSurveyPage/index.js (hunk 1 above)
Comment (AliakseiBabich) [reply src/components/Main/NewSurveyPage/index.js]: done

File: src/components/Main/NewSurveyPage/index.js (hunk 2 above)
Comment (AliakseiBabich) [reply src/components/Main/NewSurveyPage/index.js]: done, refactored a function, thanks!! =)

**Pull request merged**